src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.AspNet.Identity;
+
+namespace Microsoft.Framework.Logging
```

Comment:
There is no real conflict we don't want these to show up on all loggers unless they explicitly import an identity specific namespace. I just recommend the namespace is identity.logging instead of a random looking internal/infrastructure.